ServerlessLibraryAPI/ClientApp/src/components/DetailView/ActionBar.js

```diff
@@ -14,10 +14,21 @@ class ActionBar extends Component {
   }
 
   outboundDeployClick() {
-    libraryService.updateDownloadCount(this.props.id);
```

Comment:
how about handling the error situation inside libraryService.updateDownloadCount?